### PR TITLE
Remove XMRstak add NinjaRig

### PR DESCRIPTION
**Pull request by SoreGums**

### CODE REVIEW

File: docs/guides/mining/NinjaRig-Guide.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Would need to be compiled, instructions don't currently exist...
+
+## NinjaRig Setup and Configuration
+
+1. Unzip the file and extract the files into a new folder (Make sure your anti-virus doesn't delete the files)
+2. Open the `config.json` file with Notepad
+3. Find and change the following lines:
+
+* `"algo: "cryptonight-pico"`
```
Comment (zpalmtree) [inline docs/guides/mining/NinjaRig-Guide.md]: This seems outdated

Comment (SoreGums) [reply docs/guides/mining/NinjaRig-Guide.md]: agree :)

**Comment by SoreGums** [comment]
alrighty - that's it I think - XMRStak gone - NinjaRig added 🍾 🎆 💯

**Comment by SoreGums** [comment]
dang - didn't add a link to the sidebar...

**Comment by SoreGums** [comment]
there - that should be it.

**Comment by SoreGums** [comment]
changed algo references to `chukwa` as well

**Comment by SoreGums** [comment]
if no objections I'm a merge this in the morning (GMT+12)

**Pull request merged**